fapp/src/pages/CreateAccount/CreateAccount.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import './CreateAccount.css';
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { v4 as uuid } from 'uuid';
+
+const url = 'http://localhost:3030/accounts/'
+const user_id = 1;
+
+const CreateAccount = () => {
+    const [name, setName] = useState('');
+    const [balance, setBalance] = useState('');
+    const navigate = useNavigate();
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const new_uuid = uuid();
+        const account =
+            {
+                id: new_uuid,
+                user_id: user_id,
+                name: name,
+                balance: balance
+            };
+
+        fetch(url, {
+            method: 'POST',
+            headers: {"Content-Type": "application/json"},
+            body: JSON.stringify(account)
+        }).then(() => {
+            navigate('/accounts/' + new_uuid);
+        })
```

Comment:
Don't you think we should extract the backend logic to a separate file? Something tells me that migrating this to FireStore will be really hard if all logic will be contained in the same files as the components